CMakeLists.txt

```diff
@@ -84,8 +84,19 @@ endif()
 if(NOT TARGET FARGPARSE::fargparse)
   find_package(FARGPARSE QUIET)
 endif()
+
+
+option(USE_EXTDATA2G "Use ExtData2G" OFF)
 if(NOT TARGET YAFYAML::yafyaml)
-  find_package(YAFYAML QUIET)
+  if(USE_EXTDATA2G)
+     set (EXTDATA_TARGET "MAPL.ExtData2G" CACHE STRING "ExtData Target")
```

Comment:
I thought the plan was to build _both_ ExtData versions side by side and then have a runtime switch to control which was used?   This should not be much more trouble,  and would be more flexible.

Is there a particular concern?

Reply:
@tclune I guess my only fear was breaking UFS or whoever doesn't have yafyaml. But if we say yafyaml is required period even in MAPL 2, then you probably can get closer to a run-time switch from a build-time.

Reply:
Well, UFS will not be setting `USE_EXTDTA2G` so they will only execute the one with QUIET.   YAFAYML won't be required required until 3.0 is released.    It's just a bit redundant to check the YAFYAML target outside when you're going to be issuing FIND_PACKAGE twice inside anyway.

Reply:
Oooooohhhh! I didn't see the third call. Bad eyes.